Orange/widgets/data/owpreprocess.py

```diff
@@ -762,8 +700,8 @@ def __init__(self, parent=None, **kwargs):
         self.__scalecb.activated.connect(self.edited)
 
     def setParameters(self, params):
-        center = params.get("center", Scale.CenterMean)
-        scale = params.get("scale", Scale.ScaleBySD)
+        center = params.get("center", Scale.CenterMean - 1)
```

Comment:
The enum values are no longer comparable to integers, which IMHO is a good thing. Replaced with two short utility functions.